File: turbo/benchmarks/c/float.c
#include <stdio.h>

int main() {
    double sum = 0.0;
    double sign = 1.0;
    for (int i = 0; i < 50000000; i++) {
        sum += sign / (2 * i + 1);
        sign = -sign;
    }
    printf("%.16f\n", sum * 4.0);
    return 0;
}

File: turbo/benchmarks/rust/float.rs
fn main() {
    let mut sum = 0.0f64;
    let mut sign = 1.0f64;
    for i in 0..50_000_000i64 {
        sum += sign / (2 * i + 1) as f64;
        sign = -sign;
    }
    println!("{:.16}", sum * 4.0);
}
